lib/setup-nexus.sh

```diff
@@ -31,6 +31,9 @@ skopeo-copy "${sat_image}:${sat_version}" "${sat_image}:csm-latest"
 mkdir -p /opt/cray/etc/sat
 echo "${sat_version}" > /opt/cray/etc/sat/version
 
+# Update the cray-sat-podman package to ensureversion consistency
+zypper update cray-sat-podman
```

Comment:
Upon closer review, I think this is the wrong location for this command. It looks like lines 40-46 below are the lines that actually upload the new RPMs from this CSM release into Nexus. We want to run the `zypper update` after that happens. I'm not exactly sure which repo has the newer cray-sat-podman RPM in it, so we could just run it after all the `nexus-upload` commands.

Reply:
I took a look at Nexus on starlord, and it looks like it's included in the `csm-${RELEASE_VERSION}-embedded` repository. I think that's where packages which are also already a part of the new CSM management node images are located, if I recall correctly.

So the `zypper update` would need to be done after line 46.

Reply:
> Can you please clarify how this was tested? Was the setup-nexus.sh script just re-run after the problem was observed?

yes I observed the issue on starlord after reporting from QE while upgrading the system from 1.5 to 1.6. With the changes I reran the setup-nexus.sh script. Thats the output I added. As per your comment I addressed to have the changes after uploading the new RPMs from CSM release into Nexus.